component/libfuse/libfuse_handler.go

```diff
@@ -241,6 +241,12 @@ func (lf *Libfuse) destroyFuse() error {
 
 //export libfuse_init
 func libfuse_init(conn *C.fuse_conn_info_t, cfg *C.fuse_config_t) (res unsafe.Pointer) {
+	// fuse started processing requests, no error happened during mounting
```

Comment:
1. Any pipeline specific handling or common handlings are not done in any of our components so doing this in libfuse component itself is against the base architecture of blobfuse2.
2. this needs to be handled for both fuse2 and fuse3 solutions so doing it here is half baked solution only.

Reply:
1. IMHO, maybe we sacrificed little code cleanliness, but the benefits are significant. And I did not find any better solution for the issue.
2. Thanks for pointing it out, I've handled in fuse2 as weel.

Reply:
Let me see if we can do this in bit cleaner way. I was thinking of adding that sleep as plan-b so that in case this devlops some issue we are not completly blocked and user can move to a sleep based solution instead.

Reply:
Parent can sleep to wait, but it is just a workaround to avoid bind mount problem, which we can do it in csi driver or script rather than use `--wait-for-mount`. Actually, I think nearly no customer would use `--wait-for-mount` explicitly unless they are the expert of blobfuse. 
Also, it can't resolve the problem that when libfuse failed, blobfuse2 exit without error code and any error message in terminal or log file.

Reply:
I am not denying the fact that this PR solves problem in a better way. My concern here is just in some corner case we hit a situation where parent process is waiting for ever in some customer setup, there is no knob to disable this feature. In such case if that other sleep option is added instead of making a new release later where this option is configurable we can just suggest customer to use that mode. Default will be this signaling mode here in the PR and alternative will be to use sleep. So only on SOS basis we suggest customer to use sleep mode.

Reply:
I understand your concern, I just merged #1085 to this PR, could you take another look to see if that is ok for you?